mne/preprocessing/maxwell.py

```diff
@@ -565,8 +570,10 @@ def _overlap_projector(data_int, data_res, delta=0.02):
     C_mat = np.dot(Q_int.T, Q_res)
 
     # Compute angles between subspace and which bases to keep
-    _, S_intersect, Vh_intersect = svd(C_mat)
+    _, S_intersect, Vh_intersect = svd(C_mat, full_matrices=False)
     retain_inds = S_intersect >= (1 - delta)
+    logger.info('    Projecting out %s tSSS components for %s-%s'
+                % ((~retain_inds).sum(), tlims[0], tlims[1]))
```

Comment:
I think this is confusion brought about by bad naming. In that code, `retain_inds` refers to the vectors to keep to be projected out. It's not referring to the signal space directions that will be left over after the projection.

I'll update the variable names to be more explicit. There should be 27 and 34 components projected out in your example.